2020-10-29 12:17:48.328 - [Info] MITM Version: 1.26
2020-10-29 12:17:48.344 - [Info] Auto Access Enabled: true
2020-10-29 12:17:48.345 - [Debug] [Init] Auto Access Barrier: true
2020-10-29 12:17:48.406 - [Info] SSH man-in-the-middle server for 172.20.0.202 listening on 0.0.0.0:10001
2020-10-29 12:21:50.901 - [Debug] [Connection] Attacker connected: 193.105.134.45 | Client Identification: SSH-2.0-PuTTY_Release_0.60
2020-10-29 12:21:52.431 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "none"
2020-10-29 12:21:52.432 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 12:21:52.709 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "password"
2020-10-29 12:21:52.710 - [Debug] [Auto Access] Attacker: 193.105.134.45, Threshold: 1, Attempts: 1
2020-10-29 12:21:52.710 - [Debug] [Auto Access] Compromising the honeypot
2020-10-29 12:21:52.710 - [Debug] [Auto Access] Adding the following credentials: "123:letmein"
2020-10-29 12:21:53.026 - [Debug] [Auth] Valid credentials - Password Authentication
2020-10-29 12:21:53.026 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.202
2020-10-29 12:21:53.077 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2020-10-29 12:21:54.184 - [Debug] [Connection] Attacker closed connection
2020-10-29 12:21:54.212 - [Debug] [LXC] Container's OpenSSH server ended connection
2020-10-29 12:21:54.212 - [Debug] [LXC] Container's OpenSSH server closed connection
2020-10-29 12:25:59.323 - [Debug] [Connection] Attacker connected: 141.98.81.141 | Client Identification: SSH-2.0-Go
2020-10-29 12:26:00.163 - [Debug] [Auth] Attacker 141.98.81.141 trying to authenticate with "none"
2020-10-29 12:26:00.163 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-29 12:26:00.536 - [Debug] [Auth] Attacker 141.98.81.141 trying to authenticate with "password"
2020-10-29 12:26:00.537 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-29 12:26:00.538 - [Debug] [Auth] Attacker: 141.98.81.141 has so far made 1 attempts. Remaining: 5 attempts
2020-10-29 12:26:00.866 - [Debug] [Connection] Attacker closed the connection
